Amazon.IonObjectMapper.Test/TestObjects.cs

```diff
@@ -292,10 +292,64 @@ public override string ToString()
 
     public class Wheel
     {
+        public string Specification { get; init; }
+        public int Size { get; init; }
+        public string Brand { get; init; }
+
+        [IonField]
```

Comment:
It will be easier if you just create brand new test objects. I don't see the need for an `IonField` here.

Reply:
The `Wheel` class was not previously used by anything else in this code until this PR.
And the class has now been simplified to just have two properties.